theme/global/global.ts

```diff
@@ -20,14 +25,19 @@ export const globalCss = defineGlobalStyles({
     },
   },
   body: {
+    // scroll is needed here so homepage will be able to scroll in extension view
+    overflowY: 'auto', //  TODO: investigate this adds an un-necessary scroll bar to popout mode
     '&.no-scroll, &.no-scroll .main-content': {
       overflow: 'hidden',
     },
   },
   ...fullPageStyles,
   ...popupStyles,
-  ...popupCenterStyles,
   ...tippyStyles,
   ...radixStyles,
   ...radixTabStyles,
+  '.l-virtuoso': {
```

Comment:
Trialling this method of using a classname to share styles rather than repeated props. 

Here I am setting the dimensions for Virtuso using the classname `l-virtuoso`

I can move it into it's own file but curious to thoughts on this

Reply:
I would strongly advise against this. This creates a disconnect between the component and it's styled, and is a completely different pattern to how we're styling the rest of the app. I would be super confused trying to find where these style are added.

If you don't want to repeat these props, you could put them in an object and spread them onto the component.

Reply:
OK thanks, I'll just use a shared object